tracks/python/exercises/high-scores/mentoring.md

```diff
@@ -57,7 +57,7 @@ class HighScores(object):
         return self.sorted_scores[-1]
 
     def personal_top_three(self):
-        return self.sorted_scores[-1:-4:-1]
+        return self.sorted_scores[:-4:-1]
```

Comment:
Below this solution you probably want to mention that variation include:

```python
return self.sorted_scores[-1:-4:-1]
```

Might also be worthwhile to explain _why_ it works.

Reply:
@0thomas can you make this change?

Reply:
@SleeplessByte Yes, sure. Was a bit busy. Here comes the new commit. Suggest any words you would like to change in the reason. I tried to express it as clearly and as simple as possible.

Reply:
No problem! 

I think the new wording is just fine. We can always change in a future PR.